larq/callbacks.py

```diff
@@ -39,6 +42,14 @@ def __init__(
         self.hyperparameter = hyperparameter
         self.verbose = verbose
 
+        if update_freq not in ["epoch", "step"]:
```

Comment:
I tend to prefer the following, but not strong opinion:
```suggestion
        if update_freq not in ("epoch", "step"):
```

Reply:
Why? I have the exact opposite view but I'm curious to hear your reasoning :sweat_smile:

Reply:
Again, not a well motivated reason to change that. I think I had in mind that tuples are a bit smaller and have faster element lookup if doing `for x in sth`, but I might be totally wrong and it definitely doesn't matter here. I might be just more used to seeing a tuple version of this...
Feel free to leave it as is.